src/components/lately-history-croup/LatelyHistoryGroup.jsx

```diff
@@ -1,25 +1,42 @@
+import { useEffect, useState } from "react";
+
+import { getHistories } from "../../firebase/CRUD";
+import HistoryItem from "../shared/HistoryItem";
+
 export default function LatelyHistoryGroup() {
+  const [historyList, setHistoryList] = useState([]);
+
+  useEffect(() => {
+    chrome.runtime.sendMessage(
+      {
+        message: "userStatus",
+      },
+      (response) => {
+        if (response.message) {
+          const userToken = response.message;
+          const groupId = "New Keyword Group";
```

Comment:
P3: 공백은 관리하기가 쉽지 않을 것 같아요. Id 인 만큼 케밥케이스 등을 사용하는 것은 어떨까요?
```suggestion
          const groupId = "new-keyword-group";
```

Reply:
혹시 타이틀 등으로 사용자에게 표시되는 텍스트로 사용되는 것일 까요? 변수명과 값 사이의 관계가 궁금합니다!

Reply:
사용자가 history 버튼을 클릭하여 저장할 시, 항상 처음으로 들어갈 history group명입니다. 따라서 변수명이 아닌 default값이라고 생각해주시면 됩니다.

Reply:
의견 주신 부분에 대해 다시 생각해보니 title 필드와 group id는 구분이 필요하다고 생각되어, group id는 의견주신대로 수정하도록 하겠습니다.